### PR TITLE
avoid doctest failing when missing bcolz or pytables (hard setup)

### PR DESCRIPTION
**Pull request by juarezr**
## Changes

This is just a quality of life improvement.
Changed docstring for skipping doctests for bcolz and pytables.
It's hard to setup bcolz and pytables because they need local packages installed.
Every time that included code in docstrings they failed leading and one need to erase the docstring for getting the test to pass.

## Checklist

As there is no new features some items don't apply bellow.

* [ ] ~~Includes unit tests~~
* [x] New functions have docstrings with examples that can be run with doctest
* [ ] ~~New functions are included in API docs~~
* [x] Docstrings include notes for any changes to API or behaviour
* [x] Travis CI passes (unit tests run under Linux)
* [x] AppVeyor CI passes (unit tests run under Windows)
* [ ] ~~Unit test coverage has not decreased (see Coveralls)~~
* [ ] ~~All changes documented in docs/changes.rst~~
* [x] Ready to review and merge

### CODE REVIEW

File: petl/io/pytables.py [hunk 1]
```diff
@@ -17,35 +17,36 @@ def fromhdf5(source, where=None, name=None, condition=None,
...
-        >>> h5file.flush()
-        >>> h5file.close()
+        >>> class FooBar(tables.IsDescription): # doctest: +SKIP
+        ...     foo = tables.Int32Col(pos=0) # doctest: +SKIP
+        ...     bar = tables.StringCol(6, pos=2) # doctest: +SKIP
         >>> #
-        ... # now demonstrate use of fromhdf5
-        ... table1 = etl.fromhdf5('example.h5', '/testgroup', 'testtable')
-        >>> table1
+        >>> def setup_hdfs5_table():
```
Comment (alimanfoo) [inline petl/io/pytables.py:26]:
```suggestion
        >>> def setup_hdf5_table():
```

Avoid confusion with hdfs (something different).

File: petl/io/pytables.py [hunk 2]
```diff
@@ -17,35 +17,36 @@ def fromhdf5(source, where=None, name=None, condition=None,
...
+        ...               (2, b'qwerty'),
+        ...               (3, b'zxcvbn'))
+        ...     for row in table1[1:]:
+        ...         for i, f in enumerate(table1[0]):
+        ...             h5table.row[f] = row[i]
+        ...         h5table.row.append()
+        ...     h5file.flush()
+        ...     h5file.close()
+        >>>
+        >>> setup_hdfs5_table() # doctest: +SKIP
```
Comment (alimanfoo) [inline petl/io/pytables.py:45]:
```suggestion
        >>> setup_hdf5_table() # doctest: +SKIP
```

File: petl/io/pytables.py [hunk 3]
```diff
@@ -201,36 +202,37 @@ def fromhdf5sorted(source, where=None, name=None, sortby=None, checkCSI=False,
...
-        >>> h5table.cols.foo.create_csindex()  # CS index is required
-        0
-        >>> h5file.flush()
-        >>> h5file.close()
-        >>> #
+        >>> class FooBar(tables.IsDescription): # doctest: +SKIP
+        ...     foo = tables.Int32Col(pos=0) # doctest: +SKIP
+        ...     bar = tables.StringCol(6, pos=2) # doctest: +SKIP
+        >>>
+        >>> def setup_hdfs5_index():
```
Comment (alimanfoo) [inline petl/io/pytables.py:211]:
```suggestion
        >>> def setup_hdf5_index():
```

File: petl/io/pytables.py [hunk 4]
```diff
@@ -201,36 +202,37 @@ def fromhdf5sorted(source, where=None, name=None, sortby=None, checkCSI=False,
...
+        ...               (3, b'zxcvbn'))
+        ...     for row in table1[1:]:
+        ...         for i, f in enumerate(table1[0]):
+        ...             h5table.row[f] = row[i]
+        ...         h5table.row.append()
+        ...     h5table.cols.foo.create_csindex()  # CS index is required
+        ...     h5file.flush()
+        ...     h5file.close()
+        >>>
+        >>> setup_hdfs5_index() # doctest: +SKIP
```
Comment (alimanfoo) [inline petl/io/pytables.py:231]:
```suggestion
        >>> setup_hdf5_index() # doctest: +SKIP
```

**Comment by juarezr** [comment]
Merge, please.

**Pull request merged**